CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+## moler 1.3.0
+
+* Add entering to UNIX_LOCAL state when device is closing
+* Add devices when load config
+* Fix first moler prompt catch
+* Fix iperf mbits calculation
+* Fix iperf command
+* Add UNIX devmem cmd
+* Remove properly new line chars on some connections
```

Comment:
I would paraphrase it to be more understandable by users and markup to know new/removed/deprecated:

### Added
* force Unix device to return into local shell when device is closing (that would close ssh if was open)
* allow adding devices after initial load of configuration
* new UNIX command handled by moler: **devmem**
* iperf command refactored into iperf2 command
* fix: wrong iperf bandwidth calculation when bandwidth displayed in Kbits/sec, Mbits/sec, Gbits/sec
* fix: first moler prompt to catch
* fix: removal of new line characters on some connections

### Deprecated
* stop using Iperf command from moler.cmd.unix.iperf; use Iperf2 from moler.cmd.unix.iperf2 instead